### DISCUSSION TITLE
New commits to the branch are not reflecting in the PR page

### DISCUSSION THREAD

**Discussion by northern-mindset** [community/community#60101] [Pull Requests]
### Select Topic Area

Bug

### Body

I've pushed two commits to the PR branch few minutes ago, but they are not reflecting in the [PR](https://github.com/PureBhaktiArchive/archive-web-app/pull/363). However, I see them in the [branch](https://github.com/PureBhaktiArchive/archive-web-app/commits/daily-video). Yes, I've refreshed the PR page.

### PR view

![image](https://github.com/community/community/assets/6065071/e6028669-c085-4c3f-b2fc-340f02d2b21c)

### Branch view

![image](https://github.com/community/community/assets/6065071/363f5a48-0ca9-4796-8adf-00485d75ff66)

**Comment by dayadimova** [comment]
I have the exact same issue with one of my open PRs.

**Comment by jacobbednarz** [comment]
i managed to find a workaround from https://github.com/orgs/community/discussions/16351; switching the PR base branch and then toggling back shows them.

i did drop in a support ticket just in case it helps.

**Reply by jacobbednarz** [reply]
this has been lodged an incident. you can follow along at https://www.githubstatus.com/incidents/pr0ptqcw6q8d

**Comment by aekt** [comment]
g chi

**Comment by simas2275** [comment]
Same here. With one PR cant see new changes

**Reply by simas2275** [reply]
Seems it's fixed now

**Comment by jungbumwoo** [comment]
Same Here.

**Comment by BIMA** [comment]
Same here. The temporary solution to getting your PR updated by the latest commit is asking your colleague to push a dummy commit to your PR. It worked for me. But when I pushed my commit again afterward, the commit is not showing up again.

**Comment by jatin1213** [comment]
Same issue for me.
